lib/modules/safety/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const util = require('../../util');
+
+module.exports = function Safety(options) {
+  options = util.defaultValue(options, {});
+  options = util.permittedArgs(options, ['exec']);
+  options.exec = util.defaultValue(options.exec, () => { return new require('../../exec')(); });
+
+  const self = {};
+  self.key = 'safety';
+  self.name = 'Safety Scan';
+  self.description = 'Safety checks your installed dependencies for known security vulnerabilities.';
+  self.enabled = true;
+  let fileManager;
+
+  self.handles = function(manager) {
+    util.enforceType(manager, Object);
+    fileManager = manager;
```

Comment:
Can we add something which checks if the user has safety installed?  I appreciate it is always going to be there in docker, but some people run the cli directly.

Here is an example of how we do that in bundler-audit: https://github.com/Stono/hawkeye/blob/master/lib/modules/ruby-bundler-scan/index.js#L21